src/pf-tabs/pf-tabs.scss

```diff
@@ -0,0 +1,23 @@
+pf-tabs.nav.nav-tabs .nav.nav-tabs-pf {
+  & > li {
+    font-size: 12px;
+
+    &:first-child > a {
+      padding-left: 15px;
+
+      &:before {
+        left: 15px!important;
```

Comment:
It looks like if Patternfly removes the !important from the rule `.nav-tabs-pf>li:first-child>a:before` then this one isn't needed either.